pipeline-service/.env.example

```
@@ -11,7 +11,7 @@ PIPELINE_API_KEY=<generate-with: openssl rand -hex 32>
 
 # GitHub integration
 GITHUB_TOKEN=<github-fine-grained-token>
-GITHUB_REPOS=jackrescuer-gif/tasktime-mvp
+APP_GITHUB_REPOS=NovakPAai/tasktime-mvp
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APP_GITHUB_REPOS key should go before the GITHUB_TOKEN key

(UnorderedKey)